docs/classic-ui/images.md

```diff
@@ -254,81 +254,199 @@ The `ImageScaling` view explicitly checks the permissions of the current user.
+If we have the following two entries, the `image_srcset` outputfilter will generate one source-tag for each entry and an additional img-tag from the last entry.
 
+```json
+{
+    "medium": {
+        "title": "Large",
+        "sourceset": [
+            {
+              "scale": "mobile_crop",
+              "media": "(max-width: 768px)",
+              "additionalScales": ["mobile_crop_highres"],
+            },
+            {
+             "scale": "teaser",
+              "media": "(min-width: 769px)",
+              "additionalScales": ["large", "larger", "great", "huge"],
+            }
+        ],
+    },
+}
+```
 
-### Pixel density
+(classic-ui-images-responsive-image-support-filtering-scales)=
+#### Filtering scales
 
-Another metric is the pixel density of the user's screen.
-This metric denotes the pixel density, or resolution, of an output device in {term}`dots per inch` (DPI).
+By default for every `srcset` all available scales will be included in the `srcset`.
 
 ```json
 {
-  "scale": "huge",
-  "media": "(min-resolution: 192dpi), (-webkit-min-device-pixel-ratio: 2)"
-},
-{
-  "scale": "large"
+    "large": {
+        "title": "Large",
+        "sourceset": [
+            {"scale": "larger"},
+        ],
+    },
 }
 ```
 
-With this definition the browser will use the `huge` scale of 1600 pixels when the screen has a density of 192 DPI, also knows as `2x`.
-We use here two different media queries to also support older Safari versions.
-Mobile devices with Safari-like iPhones only support the old non-standard media query.
-If you do not care about IE support, you can use `min-resolution: 2dppx`, which is closer to `2x`.
-The most common variants are:
+to restrict the list of used scales inside of a `srcset`, one can set the `additionalScales` parameter with an array of allowed scales.
+Without this parameter all scales which are not globally excluded scales will be used.
+
+```json
+    "small": {
+        "title": "Small",
+        "sourceset": [
+            {
+              "scale": "preview",
+              "additionalScales": ["large", "larger", "great", "huge"],
+            },
+        ],
+    },
+```
 
-- `1x`: 96 DPI
-- `1.25x`: 120 DPI
-- `1.5x`: 144 DPI
-- `2x`: 192 DPI
-- `3x`: 288 DPI
+This means the generated `srcset` will contain the scales from `preview` up to `huge`, but not `mini` for example.
 
 
-### Combining viewport size and pixel density
+(classic-ui-images-responsive-image-support-picture-variant-in-editor)=
+#### Hiding a picture variant in editors
 
-It is also possible to combine the two metrics.
+It is possible to hide a picture variant in editors.
+This is useful when you want to define a picture variant to be used in templates only.
+
+```json
+    "leadimage": {
+        "title": "Lead image",
+        "sourceset": [
+            {
+              "scale": "preview",
+              "additionalScales": ["large", "larger"],
+              "hideInEditor": True,
```

Comment:
I think this edit will fix the syntax so that it is highlighted as `json`. See the preview:

https://deploy-preview-1252--6-dev-docs-plone-org.netlify.app/classic-ui/images.html#hiding-a-picture-variant-in-editors

```suggestion
              "hideInEditor": true,
```